artifactory/utils/dotnet/solution/solution.go

```diff
@@ -80,7 +80,11 @@ func (solution *solution) BuildInfo(moduleName string) (*buildinfo.BuildInfo, er
 
 		// Populate module dependencies
 		for _, dep := range dependencies {
-			module.Dependencies = append(module.Dependencies, *dep)
+			// If dependnecy has no RequestedBy field, it is not accessible in the current project -
+			// It is expected to be under a sub project.
```

Comment:
It is expected to be under a sub project.
-->
In that case, the dependency is assumed to be under a project which is referenced by this project. We therefore don't include the dependency in the build-info.